From 24839feae4177930f612850c057e589da4cba642 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 31 Jan 2018 10:30:45 +0800
Subject: [PATCH 4517/5242] MLK-19413-6 arm64: dts: fsl-imx8qm-mek: Enable
 pixel combiners

commit  d81d02c22e5f8e0e1a01714b776dcd444d59c2be from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables pixel combiners for the i.MX8qm MEK platform.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 1947e8b..8902bf3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -936,6 +936,10 @@
         status = "okay";
 };
 
+&pixel_combiner1 {
+	status = "okay";
+};
+
 &prg1 {
 	status = "okay";
 };
@@ -1000,6 +1004,10 @@
 	status = "okay";
 };
 
+&pixel_combiner2 {
+	status = "okay";
+};
+
 &prg10 {
 	status = "okay";
 };
-- 
1.7.9.5

